Cargo.toml

```diff
@@ -7,6 +7,9 @@ description = "A framework to build vhost-user backend service daemon"
 edition = "2018"
 license = "Apache-2.0"
 
+[badges]
+maintenance = { status = "deprecated" }
+
```

Comment:
We don't need this because the crate itself is not deprecated, just the repository is.

Reply:
Or maybe we should leave it here in case people where consuming the crate from the repository?